Source/Core/Common/CMakeLists.txt

```diff
@@ -222,10 +222,21 @@ target_sources(common PRIVATE
 )
 
 if(ENABLE_EGL AND EGL_FOUND)
+  find_package(Libdrm)
+  find_package(Libgbm)
+  if (LIBDRM_FOUND AND LIBGBM_FOUND)
+    target_sources(common PRIVATE
```

Comment:
This command should be indented like so:

```cmake
  target_sources(common PRIVATE
    GL/GLInterface/EGL.cpp
    GL/GLInterface/EGL.h
    GL/GLInterface/EGLDRM.cpp
    GL/GLInterface/EGLDRM.h
  )
```

Reply:
Fixed in 414cb0ec131cad6667f229c7b5b8b16969d42772